src/nft/components/details/TraitsContainer.tsx

```diff
@@ -11,7 +11,7 @@ const Grid = styled.div`
   gap: 16px;
   max-width: 780px;
 
-  @media (max-width: 960px) {
+  @media (max-width: 1080px) {
```

Comment:
`1080` works a lot better than `1024`

Reply:
doesn't it introduce a bit of inconsistency tho? I think it would be great to use same breakpoints throughout the code, even if they are not perfect in some places.